modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-sidetab/index.css

```diff
@@ -0,0 +1,68 @@
+.container-release-feature-sidetab {
+    background: var(--color-brand-primary-lighten-6, #FBFCFE);
+    height: 100%;
+    position: fixed;
+    right: -31.875rem;
+    top: 0;
+    transition: right 0.3s ease-in-out; 
+    width: 31.875rem;
+    z-index: 9999;	 
+          
+    a.link-container:hover, a.link-container.hover {
+        background-color: var(--color-action-primary-hover-10, #EDF3FE);
+        border: 0.0625rem solid var(--link-hover-color, #0053F0);
+        color: var(--link-hover-color, #0053F0);
+        text-decoration: none;
+    }
+
+    .close:hover {
+        color: var(--link-hover-color, #0053F0);		 
+    }
+  
+    .label-previous-status {
+        background-color: var(--color-neutral-2, #E1E1E4);
+        color: var(--color-neutral-7, #6C6C76);
+        text-decoration: line-through;
+    }
+  
+    .link-container {
+        background-color: var(--color-brand-primary-lighten-6, #FBFCFE);
+        border: 0.0625rem solid var(--color-brand-primary-lighten-5, #E7EFFF);
+        border-radius: 0.625rem;
+        color: var(--color-action-neutral-default, #2B3A4B);
+        font-size: 0.875rem;
+        padding: 0.75rem;
+        word-break: break-word;
+    }
+  
+    .link-container>svg {
+        fill: var(--link-color, #0B5FFF);
+        height: 0.75rem;
+        width: 0.75rem;
+    }
+  
+    .link-container span {
+        padding-left: 0.5rem;
+    } 
+    
+    .search-results .search-results-entry .search-results-entry-title {
+        color: var(--color-neutral-10, #282934);			
+        font-family: 'Source Sans Pro', sans-serif;
```

Comment:
typically I avoid setting font-family as much as possible, It should probably inherit from the root (I imagine Source Sans Pro is the default) but even if not, there should be only a few generic exceptions (ie for headings) where we set it differently. If we do need to set it, we should find the stylebook variable and use that instead of hardcoding the font name

Reply:
Yes, Source Sans Pro is the default. thanks.